systems/plants/test/evaluateNewSecondDerivativeMethod.m

```diff
@@ -0,0 +1,75 @@
+function evaluateNewSecondDerivativeMethod()
+%NOTEST
```

Comment:
I'm trying a different method to compute the gradient of a geometric Jacobian. It works and may be faster. Will pursue this later. Is it OK if I commit this?